WebInstall/explainer.md

```diff
@@ -337,21 +340,40 @@ The install capability should not work on *incognito*, and the promise should al
 
 ### Declarative install
 
-An alternate solution to allow installation of web apps is by allowing a new target type of `_install` to the HTML anchor tag.
+**An alternate solution to allow installation of web apps is by allowing a new target type of `_install` to the HTML anchor tag.**
 
 `<a href="https://airhorner.com" target="_install">honk</a>`
 
 Pros:
 * Platform fallback to navigate to the content automatically.
 * Does not need JavaScript.
 
-Cons: 
-* Takes the user out of the current context.
-* Limits the amount of information a developer can act upon that the promise provides, such as if the prompt was shown or if the origin has permissions to install apps.
-* Can become cumbersome when specifying the parameters in the tag.
-* Can't easily pass additional information like the referral-info. 
+Cons:
+* Takes the user out of the current context, providing no alternative if the use case benefits from them staying in context.
+* Limits the amount of information a developer can act upon that the promise provides, such as if the prompt was shown.
+* Developers can't modify their UX to tailor to situations where the origin permission isn't granted.
+* No support for the concept of an `install_url` (requires additional work to the HTML specification).
+* Can't easily pass additional information like the referral-info.
+
+**Another alternate solution based in a declarative approach could see the use of the [rel](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel) attribute to hint to the UA that the url in the link should be installed.**
+
+`<a href="https://airhorner.com" rel="install">honk</a>`
+
+Pros (same as above):
+* Basic platform default fallback (navigation).
+* Does not need JavaScript.
+
+Cons (same as above plus): 
+* Unintended navigation to `install_url`: If a UA doesn't implement the web install feature, and the `install_url` is used in the `href` value, it may fallback to navigate to that `install_url`, potentially leaving the end user in a blank document. (this could be resolved by creating more attributes to the `a` tag to define the additional parameters that the feature requires).
```

Comment:
Unintended navigation would still be an issue with the `target` solution above.  same with the problem with potentially needing additional new attributes to handle the other inputs (`install_url` and `manifestId`).

In general, I think that declarative install section can be simplified.  I suggest that you present both options (`target` and `rel`) first and call out all of the common issues once (as opposed to twice). Then you can additionally call out the additional complexity of how to handle an `install` `rel` with varying `target` values.

The overall message content you have in this update is sound.  I just suggest minimizing the duplication to make this easier to read/follow.